routes/api/v1/meals.js

```diff
@@ -48,4 +66,49 @@ router.get("/", function (req, res, next) {
   });
 });
 
+router.get("/:id/foods", function (req, res, next) {
+  existingMealById(req.params.id)
+  .then(mealExists => {
+    if (mealExists == false) {
+      res.setHeader("Content-Type", "application/json");
+      res.status(404).send("Meal not found");
+    } else {
+      getMealObjects(mealExists)
+      .then(meal => {
+        res.setHeader("Content-Type", "application/json");
+        res.status(200).send(JSON.stringify(meal));
+      })
+    }
+  })
+  .catch(error => {
+    res.setHeader("Content-Type", "application/json");
+    res.status(500).send({error})
+  });
+})
+
+router.post("/:id/foods/:id", function (req, res, next) {
```

Comment:
If you name the route "/:mealId/foods/:foodId" you can call those keys by using req.params.mealId and req.params.foodId.